rules-java/tests/src/test/java/org/jboss/windup/rules/apps/java/ClassMetadataTest.java

```diff
@@ -111,6 +111,14 @@ private void runTest(String inputPath) throws Exception
                 Assert.assertEquals("java.rmi.Remote", interfce.getQualifiedName());
             }
             Assert.assertEquals(1, interfaceCountFound);
+
+            // long count = 0;
+            // for (JavaMethodModel javaMethodM : javaClassModel.getMethod("testMethodDefault"))
+            // {
+            // Assert.assertEquals("testMethodDefault", javaMethodM.getMethodName());
+            // count++;
+            // }
+            // Assert.assertEquals(1, count);
         }
```

Comment:
Tried this extension of test, but it seems that we are not adding any methods into javaClassModel. Isn't that bug or am I missing something?